src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        int picker = new Random().nextInt(Color.values().length);
```

Comment:
your code creates new Random instance for each method invocation, let's move it to the class level

Reply:
Fixed